test/models/test_retinaNet_decetron2.py

```diff
@@ -107,3 +105,17 @@ def get_detectron2_models_and_inputs_batch(model_yaml_path:  str = "COCO-Detecti
     tracer = Caffe2Tracer(cfg, model, sample_inputs) #this might raise w
     onnx_model = tracer.export_onnx() #this requires pip install onnx==1.8.1
     return onnx_model, model, sample_inputs
+
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize('change_ordering', [True, False])
+@pytest.mark.parametrize('model_class', ["COCO-Detection/retinanet_R_50_FPN_1x.yaml"])
```

Comment:
If they have different trained weights you can parametrize them here also

Reply:
no other train weight (for now)